### ISSUE TITLE
How can I verify the Tesla is seeing the Jetson Xavier as a USB drive?

### ISSUE THREAD

**Issue by pfischer23** [tevora-threat/Scout#23]
TL;DR: I think I have the basic configuration correct but I can't figure out how to test if my Tesla actually wrote video to my Jetson Xavier.

I got through the setup and install mostly by reading the comments. I hooked up the Jetson Xavier (JX) to my Tesla. I saw that I had a red light on the top of the center display. I drove around for a while to collect video. 

Now what? 

I tried booting the JX at connecting it to my Ubuntu laptop. I assumed the laptop would see it as an external USB drive. It didn't. I tried plugging the JX USB cable into my Mac. Since it's running ext4 I don't expect the Mac to see it, but I expected to see a USB drive in Disk Utility. I didn't.

How can I verify that JX is actually making itself available as an external USB drive to the Tesla? Is there something specific I need to get the Ubuntu laptop to recognize it as a USB drive? Is there a way to mount the disk image inside the JX itself so I can verify that I'm actually getting the Tesla to write video to the JX?

I was a UNIX sys admin for years. Then I transitioned to network engineering. I still have some UNIX/Linux skills, but they're dated. Any help you can provide would be appreciated.

**Comment by pfischer23** [comment]
I don't know what was different before. But now when I boot the JX I have a partition named /teslaflash mounted and I see the NAME.img file in it. I can do "sudo mount NAME.img /mnt" and it mounts. The file system is empty. So I did not successfully get the Tesla to write to it. I will try mounting the JX on my Ubuntu laptop again and see if anything has changed. I'm wondering if I need to wait 5-10 minutes after the JX boots or have it connected before it powers on. I will test.

**Comment by pfischer23** [comment]
I had to read through the scripts/Xavier/l4t-usb-device-mode/nv-l4t-usb-device-mode-config.sh file. I never saw anything telling me what to name the disk image file. So I named it something like Telsa-USB-240GB-USB-Disk.img. You MUST name the disk image file /teslaflash/sdsusb.img or it will not work. Once I hard linked my original image file name to /teslaflash/sdsusb.img I rebooted and I am now able to see the USB disk image mount on my Ubuntu laptop.

**Comment by pfischer23** [comment]
I just reread the installation document. It's written clear as day in step 15 that the image file must be named sdsusb.img. PEBKAC (problem exists between keyboard and chair).
